### DISCUSSION TITLE
Quickly select inside quotes while not inside

### DISCUSSION THREAD

**Discussion by tennox** [helix-editor/helix#10003] [Q&A]
if the Cursor (`|`) is in a line somewhere near the beginning:
```
const |foo = "test";
```

is there a quick way to change inside the quotes?
I only found:
- `f"` go to first quote
- `w` / `right_arrow` to go inside (didn't even find a way to select inside quotes if you're _on_ the quote :thinking: 
- `mi"`

**Comment by kirawi** [comment]
`f"vmm`

**Reply by tennox** [reply]
thx, but tried that but the f will also _select_ till the first quote, and then pressing `v` will keep that selection when going to the second quote. (Needs extra `;` before `v`)

Also, it's around the quotes and not _inside_.

**Comment by tennox** [comment]
Okay, here is my 'solution:
```toml
[keys.normal]
C-A-o = ["move_char_right", "expand_selection"]
```

- `f"` go to first quote
- `Ctrl+Alt+o`
  - move right (to go inside quotes, as shrink_selection doesn't work)
  - expand selection to "object" (= quotes)

good enough for now :tipping_hand_man:

**Reply by tennox** [reply]
Damn, `expand_selection` only works sometimes - other times it will select around quotes too :cry:
